crates/api/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2021"
 base64 = { workspace = true }
 bytes = { workspace = true }
 futures = { workspace = true }
+mockall = { workspace = true }
```

Comment:
move down to dev dependencies?

Reply:
Alas, if this is needing to be exported it cannot go in dev-dependencies : ( -- but at the very least it should be optional and only enabled with a test_utils feature, right?